modules/recaptcha/recaptcha.go

```diff
@@ -23,11 +24,11 @@ type Response struct {
 	ErrorCodes  []string  `json:"error-codes"`
 }
 
-const apiURL = "https://www.google.com/recaptcha/api/siteverify"
+const apiURL = "/api/siteverify"
```

Comment:
Why not make the whole URL configurable?

Reply:
You'd need to have multiple URLs. The api.js and verify URL. 

Can do it if wanted

Reply:
Ah, okay. It's fine the way it is then.